Add files via upload

fabienMontagneCampusCD opened the pull request.

philipto commented:
@fabienMontagneCampusCD Please provide us with:
- the school official website URL
- the school full name
- the school street address, including the city and the country
- an URL of a page on the official website where the school offers an IT-related long-term (>1 year) course
- a proof, which shows that the school recognizes the domain you are submitting as an official email domain for the students.

fabienMontagneCampusCD commented:
Hello,

The name of the school is EPSI https://www.epsi.fr/ and more precisely https://www.epsi.fr/campus/campus-de-lille/  our address is

 91, rue Nationale, Immeuble Le Plaza, 59000 Lille, France

At this page you can found our Master program https://www.epsi.fr/programmes/ingenierie-informatique/

We having 2 kinds of mail address : @epsi.fr using by students (actualy they could use Jetbrain soft) and the teachers are using @campus-cd.net

Best regards,


Fabien MONTAGNE
Responsable pédagogique
T: 03 66 72 22 83
M: 06 47 83 08 29
***@***.***
91 rue Nationale - 59000 Lille
***@***.***D7249D.E548FC30]<http://www.epsi.fr/>
***@***.***D7249D.E548FC30]<http://www.wis-ecoles.com/>
Membres du réseau
Compétences et Développement<https://www.competences-developpement.com/>
Membres de HEP EDUCATION<https://www.hep-education.com/>
Membres de FIGS EDUCATION<http://www.figs-education.com/fr/>
Nos organismes sont qualifiés
***@***.***D7249D.E548FC30]
***@***.***D7249D.E548FC30]<https://data.sigilium.com/signatures/rcgb1n-vp8h577gs1wifrmugji-iirc/redirect_link>


Ce message et toutes les pièces jointes (ci-après le "message") sont confidentiels et établis à l'intention exclusive de ses destinataires. Toute utilisation ou diffusion non autorisée est interdite. Tout message électronique est susceptible d'altération. EPSI/WIS décline toute responsabilité au titre de ce message s'il a été altéré, déformé ou falsifié.

This message and any attachments (the "message") are confidential and intended solely for the addressees. Any unauthorized use or dissemination is prohibited. E-mails are susceptible to alteration. Neither EPSI/WIS shall be liable for the message if altered, changed or falsified
Si vous recevez ce courriel pendant vos temps non travaillés, il ne requiert ni prise de connaissance ni réponse immédiate de votre part.



De : Philip Torchinsky ***@***.***>
Envoyé : lundi 29 mars 2021 12:47
À : JetBrains/swot ***@***.***>
Cc : MONTAGNE Fabien ***@***.***>; Mention ***@***.***>
Objet : Re: [JetBrains/swot] Add files via upload (#11096)


@fabienMontagneCampusCD<https://urldefense.com/v3/__https:/github.com/fabienMontagneCampusCD__;!!FdBFez6_0g!ztu4bNf9PawE600OARcGPgUBi1eM5X5e_4eu3i-VnaKP6vJZ4mk0DkmNpiRUQaOtWw5WtQo$> Please provide us with:

  *   the school official website URL
  *   the school full name
  *   the school street address, including the city and the country
  *   an URL of a page on the official website where the school offers an IT-related long-term (>1 year) course
  *   a proof, which shows that the school recognizes the domain you are submitting as an official email domain for the students.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/JetBrains/swot/pull/11096*issuecomment-809279550__;Iw!!FdBFez6_0g!ztu4bNf9PawE600OARcGPgUBi1eM5X5e_4eu3i-VnaKP6vJZ4mk0DkmNpiRUQaOtN1fBIY8$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/ATL5CYMCEUS4JT73MRKP6MDTGBSE3ANCNFSM4ZVQ5JSA__;!!FdBFez6_0g!ztu4bNf9PawE600OARcGPgUBi1eM5X5e_4eu3i-VnaKP6vJZ4mk0DkmNpiRUQaOtZSyr8tk$>.

philipto commented:
@fabienMontagneCampusCD Thank you for the clarification. Pull request merged.

fabienMontagneCampusCD commented:
Hello,
Sorry, I have made a mistake into my previous mail : the teachers domains are campus-cd.com or reseau-cd.net (I have unfortunately concatenate the both address).

Sorry for the inconveignance.

Best regards

Fabien MONTAGNE
Responsable pédagogique
T: 03 66 72 22 83
M: 06 47 83 08 29
***@***.***
91 rue Nationale - 59000 Lille
***@***.***D724A0.B2414BF0]<http://www.epsi.fr/>
***@***.***D724A0.B2414BF0]<http://www.wis-ecoles.com/>
Membres du réseau
Compétences et Développement<https://www.competences-developpement.com/>
Membres de HEP EDUCATION<https://www.hep-education.com/>
Membres de FIGS EDUCATION<http://www.figs-education.com/fr/>
Nos organismes sont qualifiés
***@***.***D724A0.B2414BF0]
***@***.***D724A0.B2414BF0]<https://data.sigilium.com/signatures/rcgb1n-vp8h577gs1wifrmugji-iirc/redirect_link>


Ce message et toutes les pièces jointes (ci-après le "message") sont confidentiels et établis à l'intention exclusive de ses destinataires. Toute utilisation ou diffusion non autorisée est interdite. Tout message électronique est susceptible d'altération. EPSI/WIS décline toute responsabilité au titre de ce message s'il a été altéré, déformé ou falsifié.

This message and any attachments (the "message") are confidential and intended solely for the addressees. Any unauthorized use or dissemination is prohibited. E-mails are susceptible to alteration. Neither EPSI/WIS shall be liable for the message if altered, changed or falsified
Si vous recevez ce courriel pendant vos temps non travaillés, il ne requiert ni prise de connaissance ni réponse immédiate de votre part.



De : MONTAGNE Fabien
Envoyé : lundi 29 mars 2021 13:18
À : JetBrains/swot ***@***.***>; JetBrains/swot ***@***.***>
Objet : RE: [JetBrains/swot] Add files via upload (#11096)

Hello,

The name of the school is EPSI https://www.epsi.fr/ and more precisely https://www.epsi.fr/campus/campus-de-lille/  our address is

 91, rue Nationale, Immeuble Le Plaza, 59000 Lille, France

At this page you can found our Master program https://www.epsi.fr/programmes/ingenierie-informatique/

We having 2 kinds of mail address : @epsi.fr using by students (actualy they could use Jetbrain soft) and the teachers are using @campus-cd.net

Best regards,


Fabien MONTAGNE
Responsable pédagogique
T: 03 66 72 22 83
M: 06 47 83 08 29
***@***.******@***.***>
91 rue Nationale - 59000 Lille
***@***.***D724A0.B22D27B0]<http://www.epsi.fr/>
***@***.***D724A0.B22D27B0]<http://www.wis-ecoles.com/>
Membres du réseau
Compétences et Développement<https://www.competences-developpement.com/>
Membres de HEP EDUCATION<https://www.hep-education.com/>
Membres de FIGS EDUCATION<http://www.figs-education.com/fr/>
Nos organismes sont qualifiés
***@***.***D724A0.B22D27B0]
***@***.***D724A0.B22D27B0]<https://data.sigilium.com/signatures/rcgb1n-vp8h577gs1wifrmugji-iirc/redirect_link>


Ce message et toutes les pièces jointes (ci-après le "message") sont confidentiels et établis à l'intention exclusive de ses destinataires. Toute utilisation ou diffusion non autorisée est interdite. Tout message électronique est susceptible d'altération. EPSI/WIS décline toute responsabilité au titre de ce message s'il a été altéré, déformé ou falsifié.

This message and any attachments (the "message") are confidential and intended solely for the addressees. Any unauthorized use or dissemination is prohibited. E-mails are susceptible to alteration. Neither EPSI/WIS shall be liable for the message if altered, changed or falsified
Si vous recevez ce courriel pendant vos temps non travaillés, il ne requiert ni prise de connaissance ni réponse immédiate de votre part.



De : Philip Torchinsky ***@***.******@***.***>>
Envoyé : lundi 29 mars 2021 12:47
À : JetBrains/swot ***@***.******@***.***>>
Cc : MONTAGNE Fabien ***@***.******@***.***>>; Mention ***@***.******@***.***>>
Objet : Re: [JetBrains/swot] Add files via upload (#11096)


@fabienMontagneCampusCD<https://urldefense.com/v3/__https:/github.com/fabienMontagneCampusCD__;!!FdBFez6_0g!ztu4bNf9PawE600OARcGPgUBi1eM5X5e_4eu3i-VnaKP6vJZ4mk0DkmNpiRUQaOtWw5WtQo$> Please provide us with:

  *   the school official website URL
  *   the school full name
  *   the school street address, including the city and the country
  *   an URL of a page on the official website where the school offers an IT-related long-term (>1 year) course
  *   a proof, which shows that the school recognizes the domain you are submitting as an official email domain for the students.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://urldefense.com/v3/__https:/github.com/JetBrains/swot/pull/11096*issuecomment-809279550__;Iw!!FdBFez6_0g!ztu4bNf9PawE600OARcGPgUBi1eM5X5e_4eu3i-VnaKP6vJZ4mk0DkmNpiRUQaOtN1fBIY8$>, or unsubscribe<https://urldefense.com/v3/__https:/github.com/notifications/unsubscribe-auth/ATL5CYMCEUS4JT73MRKP6MDTGBSE3ANCNFSM4ZVQ5JSA__;!!FdBFez6_0g!ztu4bNf9PawE600OARcGPgUBi1eM5X5e_4eu3i-VnaKP6vJZ4mk0DkmNpiRUQaOtZSyr8tk$>.
